README.md

```diff
@@ -141,11 +142,36 @@ This is an ongoing issue: [#12](https://github.com/xuhdev/vim-latex-live-preview
 * python/dyn
 See Issue [#24](https://github.com/xuhdev/vim-latex-live-preview/issues/24), currently ```vim-latex-live-preview``` does not support ```python/dyn``` and Vim must be recompiled with Python support. 
 
+## Release History
```

Comment:
Is it really worth it given that there is a separate [CHANGELOG](https://github.com/xuhdev/vim-latex-live-preview/blob/master/CHANGELOG.md) file?